code/__DEFINES/security.dm

```diff
@@ -0,0 +1,24 @@
+#define DETSCAN_CAT_FINGERS "Prints"
```

Comment:
Should be CATEGORY.

Reply:
code/__DEFINES/construction.dm
```
//tablecrafting defines
#define CAT_NONE ""
#define CAT_WEAPONRY "Weaponry"
#define CAT_WEAPON "Weapons"
#define CAT_AMMO "Ammunition"
#define CAT_ROBOT "Robots"...```
other code uses CAT for category tho?

Reply:
Yes, a lot of our code is old and bad, anyway